lib/utils.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const buildErr = err => err ? new Error(err.message || err) : null
```

Comment:
so based on here https://github.com/bitfinexcom/bfx-facs-grc/pull/17#discussion_r1409608557 my thought is this:
const buildErr = (err) => err ? (err instanceof Error ? err : new Error(err)) : null

Reply:
well we always lost error trace so this is techinicaly an improvement